internal/query/projection/user.go

```diff
@@ -135,13 +137,15 @@ func (*userProjection) Init() *old_handler.Check {
 			handler.NewColumn(NotifyInstanceIDCol, handler.ColumnTypeText),
 			handler.NewColumn(NotifyLastEmailCol, handler.ColumnTypeText, handler.Nullable()),
 			handler.NewColumn(NotifyVerifiedEmailCol, handler.ColumnTypeText, handler.Nullable()),
+			handler.NewColumn(NotifyVerifiedEmailLowerCol, handler.ColumnTypeText, handler.Nullable()),
 			handler.NewColumn(NotifyLastPhoneCol, handler.ColumnTypeText, handler.Nullable()),
 			handler.NewColumn(NotifyVerifiedPhoneCol, handler.ColumnTypeText, handler.Nullable()),
 			handler.NewColumn(NotifyPasswordSetCol, handler.ColumnTypeBool, handler.Default(false)),
 		},
 			handler.NewPrimaryKey(NotifyInstanceIDCol, NotifyUserIDCol),
 			UserNotifySuffix,
 			handler.WithForeignKey(handler.NewForeignKeyOfPublicKeys()),
+			handler.WithIndex(handler.NewIndex("notifications_email_search", []string{NotifyInstanceIDCol, NotifyVerifiedEmailLowerCol})),
```

Comment:
```suggestion
			handler.WithIndex(handler.NewIndex("email_search", []string{NotifyInstanceIDCol, NotifyVerifiedEmailLowerCol})),
```

Reply:
the projectionname will be used as prefix anyway

Reply:
removed "users11" already, but will remove "notifications" as well, just thought that it will give additional information